.gitignore

```diff
@@ -62,3 +62,6 @@ deployments/
 
 # etherscan debugging
 etherscan_requests/
+
+# todos
+TODO.md
```

Comment:
what is this?

Reply:
I like to keep a `TODO.md` file in the repo that tracks the stuff I need to do.